digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/TierekisteriImporterOperations.scala

```diff
@@ -689,4 +689,41 @@ class EuropeanRoadTierekisteriImporter extends LinearAssetTierekisteriImporterOp
     linearAssetService.dao.insertValue(assetId, LinearAssetTypes.europeanRoadPropertyId, trAssetData.assetValue)
     println(s"Created OTH $assetName assets for ${vvhRoadlink.linkId} from TR data with assetId $assetId")
   }
-}
\ No newline at end of file
+}
+
+class SpeedLimitAssetTierekisteriImporter extends LinearAssetTierekisteriImporterOperations{
+
+  lazy val speedLimitService: SpeedLimitService = new SpeedLimitService(eventbus, vvhClient, roadLinkService)
+
+  override def typeId: Int = 20
+  override def assetName = "speedlimit"
+  override type TierekisteriClientType = TierekisteriSpeedLimitAssetClient
+  override def withDynSession[T](f: => T): T = OracleDatabase.withDynSession(f)
+  override def withDynTransaction[T](f: => T): T = OracleDatabase.withDynTransaction(f)
+
+  override val tierekisteriClient = new TierekisteriSpeedLimitAssetClient(getProperty("digiroad2.tierekisteriRestApiEndPoint"),
+    getProperty("digiroad2.tierekisteri.enabled").toBoolean,
+    HttpClientBuilder.create().build())
+
+  private def getSideCode(raSideCode: SideCode, roadSide: RoadSide): SideCode = {
```

Comment:
This method already exists on `SpeedLimitsTierekisteriImporter`

Reply:
But it is private. Or am i missing something?